### DISCUSSION TITLE
Using Radix Colors

### DISCUSSION THREAD

**Discussion by vcheeze** [sveltejs/kit#10246] [Q&A]
Hi Svelte community,

I am starting out on a new project with SvelteKit, and I want to use [Radix's color scheme](https://www.radix-ui.com/colors). I'm trying to use it simply as CSS imports, i.e. importing a stylesheet from the npm package like:

```css
@import '@radix-ui/colors/sage.css';
```

I'm doing the above in my `styles/global.scss` file, which is imported in my root `+layout.svelte`. However, the import is not being picked up, I think because it is trying to resolve `@radix-ui/` to my `styles/` folder.

Is this the default behavior with SvelteKit? Can I change this behavior with some config? I can't seem to find anywhere in the docs something related to this. Any help would be appreciated, thanks!

**Comment by david-plugge** [comment]
Try importing the css directly in the script tag of your root layout.
